web/frontend/src/components/base.vue

```diff
@@ -124,19 +125,15 @@ const digestInput = () => {
   linear.setY(linearValue);
   angular.setZ(angularValue);
 
-  const req = new baseApi.SetPowerRequest();
-  req.setName(props.name);
-  req.setLinear(linear);
-  req.setAngular(angular);
-
-  rcLogConditionally(req);
-  props.client.baseService.setPower(req, new grpc.Metadata(), (error) => {
-    displayError(error);
+  try {
+    await baseClient.setPower(linear, angular);
+  } catch (error) {
+    displayError(error as ServiceError);
 
     if (pressed.size <= 0) {
       stop();
     }
```

Comment:
confirming, do we only want this to happen on error or all the time?

Reply:
nevermind, this is beyond the scope of the PR - it's mimicking the current behavior so that's enough for now